src/transformers/modeling_reformer.py

```diff
@@ -1817,3 +1817,99 @@ def prepare_inputs_for_generation(self, input_ids, past, **kwargs):
             inputs_dict["num_hashes"] = kwargs["num_hashes"]
 
         return inputs_dict
+
+
+class ReformerClassificationHead(nn.Module):
+    """Head for sentence-level classification tasks."""
+
+    def __init__(self, config):
+        super().__init__()
+        self.dense = nn.Linear(config.hidden_size, config.hidden_size)
+        self.dropout = nn.Dropout(config.hidden_dropout_prob)
+        self.out_proj = nn.Linear(config.hidden_size, config.num_labels)
+
+    def forward(self, features, **kwargs):
+        x = features[:, 0, :]  # take <s> token (equiv. to [CLS])
```

Comment:
Could you replace all `x` and `features` with `hidden_states`? 

1) It would be great to keep reformer as memory efficient as possible -> only use one variable name in the function.
2) Even though the variable name `x` is used in other models, it's bad practice and we try to discontinue this and use more explicit variable names (also better for serach and replace)

Reply:
Noted